logical_switch.go

```diff
@@ -271,3 +271,57 @@ func (odbi *ovnDBImp) lslblistImp(lswitch string) ([]*LoadBalancer, error) {
 	}
 	return listLB, nil
 }
+
+func (odbi *ovnDBImp) lsExtIdsAddImp(lswitch string, external_ids map[string]string) (*OvnCommand, error) {
+	var operations []libovsdb.Operation
+	row := make(OVNRow)
+	row["name"] = lswitch
+	lsuuid := odbi.getRowUUID(tableLogicalSwitch, row)
+	if len(lsuuid) == 0 {
+		return nil, ErrorNotFound
+	}
+	if len(external_ids) == 0 {
```

Comment:
i'm prefer to check like external_ids == nil ?
i don't think that somebody pass non nil empty map.

Reply:
I think validation is needed for both. If external_ids == nil || len(external_ids) == 0

Reply:
Great. Thanks for explain. I am new to Go :)